modules/ROOT/pages/afm-monitoring-private-apis.adoc

```diff
@@ -19,7 +19,7 @@ You can also test public APIs using private locations. Public APIs are APIs with
 ====
 
 == Association of Private Locations with Schedules
-The tests in monitors are run on schedules. For a private location to run according to a schedule, a schedule must be associated with it. A private location can be associated with more than one schedule for more than one monitor.
+The tests in monitors are run periodically according to a selected schedule. For a private location to run according to a schedule, a schedule must be associated with it. A private location can be associated with more than one schedule for more than one monitor.
```

Comment:
This seems overly wordy, and it begs the question of public (?) location monitor testing vs. private?
Why are we specifying "private" here; don't _all_ locations need association with a schedule to run tests?
Also, this para goes from tests running according to schedule, to locations running according to schedule, which does not sound correct: this is about the test + schedule, right?

Reply:
My best guess:
```suggestion
Because monitor tests run according to specified schedules, monitoring your private location requires that location to be associated with one or more schedules (one per monitor). 
```

Reply:
Hmm, good points. I'm now thinking that the paragraph can simply be removed, as it's not presenting any new information.